README.md

```diff
@@ -133,14 +149,12 @@ tt_publisher:publish([
                      ) -> ok.
 ```
 
-`tt_publisher:publish/1` should publish on Twitter the talks. So the example invocation above should result in something like following in the Twitter account:
+`tt_publisher:publish/1` publishes on Twitter the talks. So the example invocation above result in something like following in the Twitter account:
```

Comment:
* the talks on Twitter
* the example invocation above results in something like the following in the Twitter account: